plugin/build.gradle.kts

```diff
@@ -35,19 +35,31 @@ tasks.withType<PublishTask> {
     channels(channel)
 }
 
-val langs = setOf("en", "ru", "fr", "de", "pl", "it", "zh", "ja", "uk", "el", "ro", "es", "pt", "sk", "fa", "nl")
-
 dependencies {
     compileOnly(kotlin("stdlib"))
 
     compile("org.languagetool", "languagetool-core", Versions.languageTool) {
         exclude("org.slf4j", "slf4j-api")
     }
-    for (lang in langs) {
-        compile("org.languagetool", "language-$lang", Versions.languageTool) {
-            exclude("org.slf4j", "slf4j-api")
-        }
+
+    compile("org.languagetool", "language-en", Versions.languageTool) {
+        exclude("org.slf4j", "slf4j-api")
+    }
+
+    testRuntime("org.languagetool", "language-ru", Versions.languageTool) {
+        exclude("org.slf4j", "slf4j-api")
+    }
+
+    // for PyCharm and others no Intellij Idea applications
+    compile ("org.eclipse.aether", "aether-connector-basic", "1.1.0")
+    compile ("org.eclipse.aether", "aether-transport-file", "1.1.0")
+    compile ("org.eclipse.aether", "aether-transport-http", "1.1.0") {
+        exclude("org.slf4j", "slf4j-api")
+    }
+    compile("org.apache.maven", "maven-aether-provider", "3.3.9"){
+        exclude("org.slf4j", "slf4j-api")
     }
+    // ---
```

Comment:
Huh? :)

Reply:
it's like the end of previous comment

Reply:
Hm, ok.

Reply:
You can use `//region aether-for-non-idea-ide` for it. It will also add custom folding in IDE

Reply:
Done